src/SprykerSdk/Zed/SprykGui/Business/Finder/Factory/FactoryInfoFinder.php

```diff
@@ -94,7 +94,7 @@ protected function canReflect(string $className): bool
     {
         try {
             $betterReflection = new BetterReflection();
-            $betterReflection->classReflector()->reflect($className);
+            $betterReflection->reflector()->reflectClass($className);
```

Comment:
you use roave/better-reflection package and I don't see it in composer.json

Reply:
It is provided by PHPStan